modules/kubernetes-addons/karpenter/locals.tf

```diff
@@ -26,48 +26,34 @@ locals {
   ]
 
   default_helm_config = {
-    name                       = "karpenter"
-    chart                      = "karpenter"
+    name                       = local.name
+    chart                      = local.name
     repository                 = "https://charts.karpenter.sh"
     version                    = "0.6.3"
```

Comment:
did you want to upgrade to the latest release [0.6.5](https://karpenter.sh/v0.6.5/)?

Reply:
updated